### PR TITLE
Change Koma-Script aliases

### PR DESCRIPTION
**Pull request by gucci-on-fleek**
The names of the main Koma-Script documentation files seem to have changed:
```bash
$ kpsewhich scrguien.pdf
(exit code 1)

$ kpsewhich scrguide.pdf
(exit code 1)

$ ls -l /usr/local/texlive/2022/texmf-dist/doc/latex/koma-script/
total 6472
drwxr-xr-x. 1 root root    5110 Jul 15 20:16 examples/
-rw-r--r--. 1 root root   25055 Mar 30  2009 lppl-de.txt
-rw-r--r--. 1 root root   19110 Mar 30  2009 lppl.txt
-rw-r--r--. 1 root root    9601 Jul  4 14:21 MANIFEST.md
-rw-r--r--. 1 root root   29429 Jul  4 14:21 README.md
-rw-r--r--. 1 root root    2425 Jul  4 14:21 scraddr.html
-rw-r--r--. 1 root root    2564 Jul  4 14:21 scrartcl.html
-rw-r--r--. 1 root root    2570 Jul  4 14:21 scrarticle.html
-rw-r--r--. 1 root root    2439 Jul  4 14:21 scrbase.html
-rw-r--r--. 1 root root    2561 Jul  4 14:21 scrbook.html
-rw-r--r--. 1 root root    2383 Jul  4 14:21 scrdate.html
-rw-r--r--. 1 root root    2793 Jul  4 14:21 scrextend.html
-rw-r--r--. 1 root root 3257748 Jul  4 14:21 scrguide-de.pdf
-rw-r--r--. 1 root root 3163933 Jul  4 14:21 scrguide-en.pdf
-rw-r--r--. 1 root root    2447 Jul  4 14:21 scrhack.html
-rw-r--r--. 1 root root    2465 Jul  4 14:21 scrjura.html
-rw-r--r--. 1 root root    2713 Jul  4 14:21 scrkbase.html
-rw-r--r--. 1 root root    2492 Jul  4 14:21 scrlayer.html
-rw-r--r--. 1 root root    2463 Jul  4 14:21 scrlayer-notecolumn.html
-rw-r--r--. 1 root root    2478 Jul  4 14:21 scrlayer-scrpage.html
-rw-r--r--. 1 root root    2565 Jul  4 14:21 scrletter.html
-rw-r--r--. 1 root root    2647 Jul  4 14:21 scrlfile-hook-3.34.html
-rw-r--r--. 1 root root    2602 Jul  4 14:21 scrlfile-hook.html
-rw-r--r--. 1 root root    2484 Jul  4 14:21 scrlfile.html
-rw-r--r--. 1 root root    2683 Jul  4 14:21 scrlfile-patcholdlatex.html
-rw-r--r--. 1 root root    2459 Jul  4 14:21 scrlogo.html
-rw-r--r--. 1 root root    2562 Jul  4 14:21 scrlttr2.html
-rw-r--r--. 1 root root    2567 Jul  4 14:21 scrreport.html
-rw-r--r--. 1 root root    2564 Jul  4 14:21 scrreprt.html
-rw-r--r--. 1 root root    2379 Jul  4 14:21 scrtime.html
-rw-r--r--. 1 root root    2470 Jul  4 14:21 scrwfile.html
-rw-r--r--. 1 root root    2446 Jul  4 14:21 tocbasic.html
-rw-r--r--. 1 root root    2448 Jul  4 14:21 typearea.html
```

### CODE REVIEW

File: texdoc.cnf [hunk 1]
```diff
@@ -369,18 +369,18 @@ alias amsproc-dev = amsclass
...
-alias scrartcl-de = scrguide
-alias scrbook-de = scrguide
-alias scrreprt-de = scrguide
-alias typearea-de = scrguide
+alias koma = scrguide-en
+alias koma-script = scrguide-en
+alias scrartcl = scrguide-en
+alias scrbook = scrguide-en
+alias scrreprt = scrguide-en
+alias typearea = scrguide-en
```
Comment (muzimuzhi) [inline texdoc.cnf:377]: Perhaps the suffix `-en` can be dropped here so that `texdoc` can use user's language setting?

Comment (gucci-on-fleek) [reply texdoc.cnf:377]: Without the `-en`, `texdoc koma` opens `scrguide-de.pdf` on my system, with `LC_ALL=en_CA` and `LANG=en_CA.UTF-8`. I don't really understand how aliases work though, so there may perhaps be another way.

Comment (wtsnjp) [reply texdoc.cnf:377]: I think with `-en` is just fine for this case. We may want to show English version first for all locales except `de`.

**Comment by wtsnjp** [comment]
Thanks. Merged.

**Pull request merged**